src/main/java/EmptyTaskException.java

```diff
@@ -0,0 +1,14 @@
+public class EmptyTaskException extends Exception {
```

Comment:
This class is well written! Good job!
Perhaps add a short JavaDoc for this class so its easier for you to revisit next time